sklearn/ensemble/_hist_gradient_boosting/binning.py

```diff
@@ -130,11 +130,15 @@ class _BinMapper(TransformerMixin, BaseEstimator):
         it is always equal to ``n_bins - 1``. Note that if ``n_bins_missing_``
         is less than ``n_bins - 1`` for a given feature, then there are
         empty (and unused) bins.
+    n_threads : int, default=1
+       The number of OpenMP threads.
     """
-    def __init__(self, n_bins=256, subsample=int(2e5), random_state=None):
+    def __init__(self, n_bins=256, subsample=int(2e5), random_state=None,
+                 n_threads=1):
```

Comment:
This is to avoid to duplicate the validation of `n_jobs`/`n_threads`. It is done once in `HistGradientBoosting`. Then all private subclass expect a positive integer (since num_threads in prange will expect so). This also the reason to use the naming `n_thread` to specify that we handle OpenMP threads.